files/KoreBuild/scripts/KoreBuild.psm1

```diff
@@ -197,12 +197,14 @@ function Install-Tools(
         Write-Warning "dotnet shared runtime channel overridden by KOREBUILD_DOTNET_SHARED_RUNTIME_CHANNEL"
     }
     if ($env:KOREBUILD_DOTNET_SHARED_RUNTIME_VERSION) {
-        $runtimeVersion = $env:KOREBUILD_DOTNET_SHARED_RUNTIME_VERSION
+        $runtimeVersions = $env:KOREBUILD_DOTNET_SHARED_RUNTIME_VERSION
```

Comment:
Environment variables are always a single string. We should probably split on some separator, such as
`$env:KOREBUILD_DOTNET_SHARED_RUNTIME_VERSION -split ';'`

Reply:
Forgot to click submit on the my comments earlier. Do we still use these switches? If not, might be nicer to just get rid of these entirely. I'd certainly not want to support adding more features (multiple versions for instance) to it though.

Reply:
I'd love to get rid of these altogether. 👍 We do have a few edge cases where we need them, but otherwise I'm in favor of dropping these.